### PR TITLE
DataSession code is incorrect as it overides backend, operations wizard has nonused var

### PR DESCRIPTION
**Pull request by LTDakin**
Some code cleaning as I'm researching stories

- displayImages not used anywhere
- moved the ref to be at the top with the rest of the refs
- the if check is overwriting the source attribute of images, maybe it was for when the site was first starting up but we need source to be datalab as well for future stories

Tested to make sure everything still works

### CODE REVIEW

File: src/components/DataSession/DataSession.vue [hunk 1]
```diff
@@ -85,11 +85,6 @@ function deleteOperation(operationID) {
 
 async function addOperation(operationDefinition) {
   const url = dataSessionsUrl + props.data.id + '/operations/'
-  if ('input_files' in operationDefinition.input_data) {
-    for (const image of operationDefinition.input_data.input_files) {
-      image.source = 'archive'
```
Comment (capetillo) [inline src/components/DataSession/DataSession.vue:90]: Can't remember so: does `image.source` have an assigned value prior to this?

Comment (LTDakin) [reply src/components/DataSession/DataSession.vue:90]: yeah its gets that property when its sent back from the archive or datalab, however this code was overwriting the image.source = 'datalab' with 'archive' which was wrong

**Review by capetillo — COMMENTED** [review]
Left a question : ) so if `image.source` comes with a value, then all is good. But if it doesn't, there is an existence check below that could cause errors!

File: src/components/DataSession/DataSession.vue (hunk 1 above)
Comment (capetillo) [reply src/components/DataSession/DataSession.vue:90]: gotcha! then LGTM 🎉

Comment (LTDakin) [reply src/components/DataSession/DataSession.vue:90]: we may need to still add a method for patching in missing sources but this code was arbitrarily replacing all image sources with 'archive' which isn't what we want to happen

**Review by capetillo — APPROVED** [review]
🤠

**Pull request merged**